projects/sunbird-video-player/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@project-sunbird/sunbird-video-player-v9",
-  "version": "4.10.1",
+  "version": "4.10.2",
   "peerDependencies": {
     "@angular/common": ">= 8.0.0",
     "@angular/core": ">= 8.0.0",
     "@types/video.js": "7.5.5",
-    "video.js": "7.4.1",
```

Comment:
What is the reason to change the video js version and is it tested with the existing video plugins?

Reply:
When we were trying to implement the non-fullscreen mode for video player ,
Video fullscreen maximize and minimize via icon was not working 
We have tested plugins for resolution & speed .